CONTRIBUTING.md

```diff
@@ -33,14 +33,14 @@ See [SECURITY.md](SECURITY.md#reporting-security-issues) for instructions.
 
 ## Documentation
 
-The Splunk Observability documentation is hosted on <https://docs.splunk.com/Observability>,
-which contains all the prescriptive guidance for Splunk Observability products.
+The Splunk Observability documentation is hosted on https://docs.splunk.com/Observability,
+which contains all the prescriptive guidance for Splunk Observability products. 
 Prescriptive guidance consists of step-by-step instructions, conceptual material,
-and decision support for customers. Reference documentation and development
+and decision support for customers. Reference documentation and development 
 documentation is hosted on this repository.
 
-You can send feedback about Splunk Observability docs by clicking the Feedback
-button on any of our documentation pages.
+To contribute documentation for this project, open a pull request in the 
```

Comment:
"this project" read like this repo to me and got me confused for a bit, perhaps be explicit to get clarity at first reading, something like
```suggestion
To contribute to the Splunk Observability documentation, open a pull request in the 
```